### PR TITLE
Feature training transparency

### PR DESCRIPTION
**Pull request by clennan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,37 @@ evaluater = Evaluation(image_dir=dp.image_dir, job_dir=dp.job_dir)
 evaluater.run()
```
Comment (gunarmaiwald) [inline README.md]: chris, could you rename 'evaluater' into 'evaluator'?

Comment (clennan) [reply README.md]: Done.

File: imageatm/components/training.py [hunk 2]
```diff
@@ -137,18 +155,45 @@ def _fit_model(self):
             save_weights_only=False,
         )
 
+        def _set_patience():
+            """Adjust patience for early stopping and learning rate schedule
```
Comment (gosia-malgosia) [inline imageatm/components/training.py]: just a thought: is the mean n per class the best indicator or would it be more beneficial to look at the largest/smallest class?

Comment (clennan) [reply imageatm/components/training.py]: With a smallest class approach a single minority class could slow down training significantly with no assurances that model performance benefits from more epochs - so I believe the mean class size is the best we can do without further experimentation.

**Comment by datitran** [comment]
Hmm it seems that the tests are failing?

**Pull request merged**